README.md

```diff
@@ -32,6 +32,18 @@ bash dockler.sh
 # Step 3. Start the crawl job
 /data/sparkler/bin/sparkler.sh crawl -id 1 -tn 100 -i 2     # id=1, top 100 URLs, do -i=2 iterations
 ```
+### Running Sparkler with seed urls file: 
+```bash
+1. Follow Steps 0-1
+2. Create a file name seed-urls.txt using Emacs editor as follows:     
+       a. emacs sparkler/bin/seed-urls.txt 
+       b. copy paste your urls 
+       c. Ctrl+x Ctrl+s to save  
+       d. Ctrl+x Ctrl+c to quit the editor [Reference: http://mally.stanford.edu/~sr/computing/emacs.html]
```

Comment:
@prenastro 
Just an FYI. you can also do
```bash
echo "http://example.com" >> seedfile.txt
```
to create a seed file. No editor needed!

Reply:
I did not know this. Thanks! 👍 :)